.github/workflows/codespell.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Codespell
```

Comment:
The three dashes (`---`) used to indicate the start of a YAML document should not be mandatory.
```suggestion
name: Codespell
```